src/Application/src/RazorPagesTestSample/Data/Message.cs

```diff
@@ -9,7 +30,7 @@ public class Message
 
         [Required]
         [DataType(DataType.Text)]
-        [StringLength(200, ErrorMessage = "There's a 200 character limit on messages. Please shorten your message.")]
+        [StringLength(250, ErrorMessage = "There's a 250 character limit on messages. Please shorten your message.")]
```

Comment:
There were some tests related to this can you check once if they need update as per the change.